.github/ISSUE_TEMPLATE/new-dbt-model.md

```diff
@@ -35,14 +38,18 @@ Otherwise, delete it in favor of the long checklist in the following section.)_
 - [ ] Confirm that a subdirectory for this model's database exists in
   the `dbt/models/` directory, and if not, create one, add a new `schema.yml`
   file, and update `dbt_project.yml` to document the `+schema`
-- [ ] Define the SQL query that creates the model in the model subdirectory,
-  following any existing file naming schema
-- [ ] Optionally configure model materialization within the query file
+- [ ] Define the SQL query or Python script that creates the model in the model
+  subdirectory, following any existing file naming schema
+- [ ] Use `source()` and `ref()` to reference other models where possible
```

Comment:
This line was present in the expanded checklist but missing from the condensed checklist, so I added it in.